sycl/source/detail/config.hpp

```diff
@@ -15,6 +15,7 @@
 #include <algorithm>
 #include <array>
 #include <cstdlib>
+#include <cstring>
```

Comment:
```suggestion
#include <string>
```
I don't see usages on `cstring` functions, but `std::string` is used here. Should `string` be included instead?

Reply:
done.